components/director/internal/domain/labeldef/resolver.go

```diff
@@ -127,15 +129,15 @@ func (r *Resolver) LabelDefinition(ctx context.Context, key string) (*graphql.La
 	def, err := r.srv.Get(ctx, tnt, key)
 
 	if err != nil {
+		if apperrors.IsNotFoundError(err) {
+			return nil, nil
+		}
 		return nil, errors.Wrap(err, "while getting Label Definition")
 	}
 	if err := tx.Commit(); err != nil {
 		return nil, errors.Wrap(err, "while committing transaction")
 	}
 
-	if def == nil {
```

Comment:
I think it is still safer to check if def is not nil, as we return pointer from the service. I know, this situation shouldn't happen, but still :P return new `apperror.NotFound` error, just in case